eslintPlugin/src/noAnimatedStyleToNonAnimatedComponent.ts

```diff
@@ -0,0 +1,155 @@
+import { AST_NODE_TYPES, TSESLint, TSESTree } from '@typescript-eslint/utils';
+
+const rule: TSESLint.RuleModule<'animatedStyle' | 'sharedValue'> = {
+  create: function (context) {
+    return {
+      JSXOpeningElement(node: TSESTree.JSXOpeningElement) {
+        if (node.name.type === AST_NODE_TYPES.JSXMemberExpression) return; // Property-like namespace syntax <Animated.View>
+        if (node.name.type === AST_NODE_TYPES.JSXNamespacedName) return; // XML-based namespace syntax: <Animated:View>
```

Comment:
Is this actually the case in React Native? 👀

Reply:
It's not, React components cannot include JSX namespace names.
Yet, I want to keep this condition to avoid unncecessary risk.

Reply:
Add a comment about it here then please.